src/scales/scale.ts

```diff
@@ -93,7 +89,7 @@ module Plottable {
      * @returns {R} The range value corresponding to the supplied domain value.
      */
     public scale(value: D): R {
-      return this._d3Scale(value);
+      throw new Error("subclasses should override functionality");
```

Comment:
capitalize. Now we're not consistent, everywhere above it was stated the function name in the error